config/locales/zh_CN.yml

```diff
@@ -0,0 +1,40 @@
+zh_CN:
+  errors:
+    messages:
+      taken_in_past: '曾被使用过。'
+      equal_to_current_password: '必须与当前密码不同。'
+      password_complexity:
+        digit:
+          one: 必须包含至少1个数字
+          other: 必须包含至少%{count}个数字
+        lower:
+          one: 必须包含至少1个小写字母
+          other: 必须包含至少%{count}个小写字母
+        symbol:
+          one: 必须包含至少1个标点符号
+          other: 必须包含至少%{count}个标点符号
+        upper:
+          one: 必须包含至少1个大写字母
+          other: 必须包含至少%{count}个大写字母
+  devise:
+    invalid_captcha: '无效验证码'
```

Comment:
Pretty sure this is another issue of me missing nuance trying to use Google Translate. Google translates this to "Invalid verification code." 

This entry deals with more with [CAPTCHAs](https://en.wikipedia.org/wiki/CAPTCHA) and not the verification code. 

Is there anything that more directly ties to CAPTCHA, or is this the best phrase? 

I'm checking because we have the idea of verification codes as a part of our `ParanoidVerificatoin` module and have some translations for them. I wanted to see if we could have them be more distinguishable if possible.

Reply:
The direct, word by word translation of "验证码" is verification code, but in Chinese websites, it is used for "CAPTCHAs" and "verification code". 

Since the other word for CAPTCHAs, "全自动区分计算机和人类的图灵测试" , is the direct translatin of "Completely Automated Public Turing test to tell Computers and Humans Apart", "验证码" is the best phrase for "CAPTCHAs" and "verification code". 


CAPTCHAs:
![图片](https://user-images.githubusercontent.com/15083013/95480186-762ab780-0940-11eb-8662-28934bd2765f.png)
Verification code:
![图片](https://user-images.githubusercontent.com/15083013/95480848-387a5e80-0941-11eb-95ca-8f7b943f2941.png)

Reply:
Thank you for the explanation and screenshot examples 👍